MAUI/TextInputLayout/States-And-Colors.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title: States and Colors in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about States and Colors support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# States and Colors in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Use the visual state manager to change the `Stroke` properties based on the visual states set from code. The applicable visual states are normal, focused, and error.
```

Comment:
Use the visual state manager to change the `Stroke` properties based on the visual states set from the code. The applicable visual states are normal, focused, and error.

Reply:
Addressed the changes